app/src/main/java/com/hover/stax/bounties/BountyViewModel.java

```diff
@@ -81,4 +91,13 @@ private void makeBounties(List<HoverAction> actions, List<StaxTransaction> trans
 		}
 		bountyList.setValue(bounties);
 	}
+
+	public void setSimPresentBounty(Bounty b) {
```

Comment:
This won't actually catch SIM changes. Need to call `Hover.updateSimInfo(context);` to ensure the DB gets updated. The sim update worker only runs every 15 minutes without this manual call. See ChannelDropdown for example.